memoized_lcsh.json

```diff
@@ -408,5 +408,7 @@
   "Illinois--Chicago Metropolitan Area": "https://id.loc.gov/authorities/subjects/sh2009008684-781",
   "Latin America": "http://id.loc.gov/authorities/subjects/sh85074879",
   "Viral Proteins": "https://id.loc.gov/authorities/subjects/sh94006884",
-  "Cytology": "https://id.loc.gov/authorities/subjects/sh99005436"
+  "Cytology": "https://id.loc.gov/authorities/subjects/sh99005436",
```

Comment:
Same as `memoized_lcnaf`. Not sure why this wasn't committed, but it seems fine to just throw it in here.